core/events/producer.py

```diff
@@ -84,3 +107,5 @@ def publish_log(self, log: str | dict):
 
 
 producer = EventProducer()
+
+producer = EventProducer()
```

Comment:
Duplicate line!

Reply:
fixed in f5a7fd9fdb02161346b5b6a6d5f7022d7ba33fdc